experanto/experiment.py

```diff
@@ -5,74 +5,73 @@
 from .interpolators import Interpolator
 import re
 
-Interval = namedtuple("Interval", ["start", "end"])
 
+class Experiment:
 
-class Experiment(Sequence):
-
-    def __init__(self, root_folder: str, sampling_rate: float) -> None:
+    def __init__(self, root_folder: str) -> None:
         self.root_folder = Path(root_folder)
-        self.sampling_rate = sampling_rate
         self._devices = dict()
         self.device_names = list()
```

Comment:
That seems redundant and a possible cause for errors. I would suggest to make it a property:

```python
@property
def device_names(self):
    return tuple(self._devices.keys())
```